lib/omnibus/builder.rb

```diff
@@ -644,6 +644,39 @@ def sync(source, destination, options = {})
     expose :sync
 
     #
+    # Helper method to update config_guess in the software's source
+    # directory.
+    # @param [Hash] options
+    #   Supported options are:
+    #     target [String] subdirectory under the software source to copy
+    #       config.guess.to. Default: "."
+    #     install [Array<Symbol>] parts of config.guess to copy.
+    #       Default: [:config_guess, :config_sub]
+    def update_config_guess(target: ".", install: [:config_guess, :config_sub])
+      build_commands << BuildCommand.new("update_config_guess `target: #{target} install: #{install.inspect}'") do
+        destination = File.join(software.project_dir, target)
+        FileUtils.mkdir_p(destination)
+
+        config_guess_dir = FileSyncer.glob("#{Config.source_dir}/*/").select do |d|
+          d =~ /config_guess/
```

Comment:
Why are we doing this via a regex rather than as part of the glob?  Also, what do we expect these to be named.  The error messages below seem to indicate that the directory will be named `config_guess-SOMETHING` but this would pick up much more than that.